ui/plugin-system/src/components/PanelSpecEditor/PanelSpecEditor.tsx

```diff
@@ -63,8 +96,25 @@ export function PanelSpecEditor(props: PanelSpecEditorProps) {
     );
   }
 
-  // always show json editor by default
-  tabs.push({ label: 'JSON', content: <JSONEditor value={panelDefinition} onChange={onJSONChange} /> });
+  // Always show JSON editor
+  tabs.push({ label: JSON_TAB_LABEL, content: <JSONEditor value={panelDefinition} onChange={onJSONChange} /> });
+
+  // Keep  track of active tab
+  const defaultTab = tabs.find((t) => t.label === QUERY_TAB_LABEL) ? QUERY_TAB_LABEL : GENERAL_TAB_LABEL;
```

Comment:
Since we can’t drag & drop panels between panel groups w/o using the ‘Group’ dropdown, I worry people will have a harder time finding that workaround now

Reply:
I also feel like it might be important to be able to see the query when switching panel types

Reply:
Yeah the idea for now is to just default to the second tab.

For panels with queries, the second tab is the "Query" tab.
For panels without queries, the second tab is the first custom tab, or JSON if they don't specify anything. 

Yeah, if we wanted panel plugin authors to be able to specify a default tab that wasn't the first custom tab, we would probably want to add a field for that. 

For now / in practice, I'm guessing that the first custom tab should be the most important tab. And if it's not, they should probably switch the order of their tabs :)

Reply:
sounds good to me!